scaling: 1000.0
smoothing: 10
resolutionIndex: 1
gainIndex: 2
settlingFactor:  0
differential: True

right_m: 1.0
right_b: 0.0690979924947

left_m: 1.0
left_b: 0.130114241402